rules_gapic/gapic.bzl

```diff
@@ -40,11 +40,13 @@ def _gapic_srcjar_impl(ctx):
         _set_args(attr.service_yaml, "--service_yaml=", arguments, inputs)
         _set_args(attr.package_yaml2, "--package_yaml2=", arguments, inputs)
         _set_args(attr.grpc_service_config, "--grpc_service_config=", arguments, inputs)
+        _set_args(attr.transport_protocol, "--transport_protocol=", arguments)
```

Comment:
Could we please document the valid values of `transport_protocol` somewhere?

Reply:
I updated the code to match it the main regapic spec. Those values are documented there (and this argument was changed from transport_protocol to just transport). The values are `grpc`, `rest`, `grpc+rest`. This implementation does not support `grpc+rest` because it is not needed for GCE (the main thing we need now), and investing time end effort into supporting the proper grpc+rest in monolith is probably not worth it.